java/statestore/src/test/java/sleeper/statestore/dynamodb/DynamoDBFileReferenceStoreIT.java

```diff
@@ -141,6 +141,26 @@ void shouldAddFileSplitOverTwoPartitions() throws Exception {
                     withLastUpdate(updateTime, leftFile),
                     withLastUpdate(updateTime, rightFile));
         }
+
+        @Test
+        void shouldReturnActiveFilesOrderedByPartitionIdThenFilename() throws Exception {
```

Comment:
This test isn't in the other reference store test classes. Where should it be? It's in the HandleIngest nested class but doesn't seem to be related to ingest.

Can the name be changed? There's no assertion on ordering.